packages/utilities/src/initials.test.ts

```diff
@@ -27,4 +35,28 @@ describe('getInitials', () => {
     let result = getInitials('Kat Larrson', true);
     expect(result).to.equal('LK');
   });
+
+  it('calculates an expected initials for Arabic names', () => {
+    let result = getInitials('خسرو رحیمی', true);
+    expect(result).to.equal('ی');
+  });
+
+  it('calculates an expected initials for Chinese names', () => {
+    let result = getInitials('桂英', false);
```

Comment:
Very common for Chinese to have 3 characters, sometimes even 4. It would be great to have one such test, just to ensure for instance that no more than 2 characters are returned.